pyproject.toml

```diff
@@ -17,9 +20,11 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.ruff]
 select = ["ALL"]
+ignore = ["ANN101", "ANN102"]  # Type annotations for 'self' and 'cls'
```

Comment:
Do these not work properly?

Reply:
They are not auto-fixable. I do not always want to add `self: "Foo"` or `cls: type["Foo"]` (or rather `cls: Self`) to remove the warning. Therefore, I put them into the set of ignored rules.